webrtc.html

```diff
@@ -8225,9 +8231,43 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <code>"new"</code>.</p>
           </li>
           <li>
+            <p>Let <var>transport</var> have a <dfn>[[\MaxMessageSize]]</dfn>
+            internal slot and run the steps labeled <a>update the data max
+            message size</a> to initialize it.</p>
+          </li>
+          <li>
             <p>Return <var>transport</var>.</p>
           </li>
         </ol>
+        <p>To <dfn>update the data max message size</dfn> of an <code>
+        <a>RTCSctpTransport</a></code> run the following
+        steps:</p>
+        <ol>
+          <li>
+            <p>Let <var>transport</var> be the <code><a>RTCSctpTransport</a>
+            </code> object to be updated.</p>
+          </li>
+          <li>
+            <p>Let <var>remoteMaxMessageSize</var> be the value of the
+            "max-message-size" SDP attribute read from the remote description,
+            as described in [[!SCTP-SDP]], or 65535 if the attribute is
+            missing.</p>
+          </li>
+          <li>
+            <p>Let <var>canSendSize</var> be the number of bytes that this
+            client can send (i.e. the size of the local send buffer) or 0 if
+            the implementation can handle messages of any size.</p>
```

Comment:
Note: In the W3C API, sending messages of any size is impossible at the moment (the API lacks the possibility to send partial messages) but I think it's good to have it in there already to be future-proof (and as a note for implementations outside of the browser).